4_gap_analysis/src/Con_byState.R

```diff
@@ -115,29 +123,34 @@ sample_rates <- samples_per_year %>%
     MeanSamplesPerYear=mean(NumSamplesPerYear),
     MinSamplesPerYear=min(NumSamplesPerYear),
     MaxSamplesPerYear=max(NumSamplesPerYear),
-    MedianSamplesPerYear=median(NumSamplesPerYear))
+    MedianSamplesPerYear=median(NumSamplesPerYear),
+    NoYears=n_distinct(Year))
 
 # split into scenarios
 # Do some more work here
-scenario_empty <- sample_rates %>%
-  filter(MinNumConstituents == 3, MinSamplesPerYear >= 12)
 scenario_A <- sample_rates %>%
-  filter(MinNumConstituents == 2, MinSamplesPerYear >= 12)
+  filter(MinNumConstituents == 3, MinSamplesPerYear >= 12, NoYears>4)
 scenario_B <- sample_rates %>%
-  filter(MinNumConstituents == 1, MinSamplesPerYear >= 12)
+  filter(MinNumConstituents < 3, MinSamplesPerYear >= 12, NoYears>4)
 scenario_C <- sample_rates %>%
-  filter(MinNumConstituents == 2, MinSamplesPerYear >= 6)
+  filter(MinNumConstituents == 3, MinSamplesPerYear >= 6 & MinSamplesPerYear < 12, NoYears>4)
+scenario_D <- sample_rates %>%
+  filter(MinNumConstituents < 3, MinSamplesPerYear >= 6 & MinSamplesPerYear < 12, NoYears>4)
+scenario_E <- sample_rates %>%
+  filter(MinNumConstituents == 3, MinSamplesPerYear >= 4 & MinSamplesPerYear < 6, NoYears>4)
+
+siteInfo<-rbind(scenario_A,scenario_B,scenario_C,scenario_D,scenario_E)
+siteInfo$cat<-ifelse(siteInfo$MonitoringLocationIdentifier %in% scenario_A$MonitoringLocationIdentifier,'A) 3 constituents (con.), >= 12 samples/year',
+                     ifelse(siteInfo$MonitoringLocationIdentifier %in% scenario_B$MonitoringLocationIdentifier,'B) 1-2 con., >= 12 samples/year',
+                            ifelse(siteInfo$MonitoringLocationIdentifier %in% scenario_C$MonitoringLocationIdentifier,'C) 3 con. 6-12 samples/year',
+                                   ifelse(siteInfo$MonitoringLocationIdentifier %in% scenario_D$MonitoringLocationIdentifier,'D) 1-2 con. 6-12 samples/year',
+                                          ifelse(siteInfo$MonitoringLocationIdentifier %in% scenario_E$MonitoringLocationIdentifier,'E) 3 con. 4-6 samples/year',NA)
+                            ))))
 
-siteInfo<-rbind(scenario_A,scenario_B,scenario_C)
-siteInfo$cat<-ifelse(siteInfo$MonitoringLocationIdentifier %in% scenario_A$MonitoringLocationIdentifier,"A) 2 cons., >=12 samples/year",
-                     ifelse(siteInfo$MonitoringLocationIdentifier %in% scenario_B$MonitoringLocationIdentifier,"B) 1 cons., >=12 samples/year",
-                            ifelse(siteInfo$MonitoringLocationIdentifier %in% scenario_C$MonitoringLocationIdentifier,"C) 2 cons. >=6 samples/year","D) <6 samples/year")))
 
-state<-gsub("_"," ",state)
-stateAb<-state.abb[grep(state, state.name)]
 
-write_feather(siteInfo,paste("andy/",stateAb,"_samples.feather",sep=""))
-write.csv(siteInfo,paste("andy/",stateAb,"_samples.csv",sep=""),row.names=F,quote=F)
+write_feather(siteInfo,paste('andy/',stateAb,'_samples.feather',sep=''))
+write.csv(siteInfo,paste('andy/',stateAb,'_samples.csv',sep=''),row.names=F,quote=F)
```

Comment:
Maybe think about moving all the output files in `andy/` into `4_gap_analysis/out/` or similar? Some folder whose name says more about the types of files it contains?